core/src/main/java/org/opensearch/sql/data/model/ExprDatetimeValue.java

```diff
@@ -33,7 +33,7 @@ public class ExprDatetimeValue extends AbstractExprValue {
 
   static {
     FORMATTER_VARIABLE_NANOS = new DateTimeFormatterBuilder()
-        .appendPattern("yyyy-MM-dd HH:mm:ss")
+        .appendPattern("uuuu-MM-dd HH:mm:ss[xxx]")
```

Comment:
I don't understand the difference between `uuuu` and `yyyy` for year.  They seem the same to me according to https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatterBuilder.html#appendPattern-java.lang.String-

Reply:
https://howtodoinjava.com/java/date-time/resolverstyle-strict-date-parsing/
https://stackoverflow.com/a/30478777
It was recommended to use it multiple places in regards to strict date parsing. yyyy has to do with eras.